src/main/res/values/themes.xml

```diff
@@ -15,11 +15,10 @@
             <item name="colorControlHighlight">@color/simple_alpha33</item>
         <item name="alertDialogTheme">@style/AppCompatAlertDialogStyle</item>
         <item name="actionOverflowButtonStyle">@style/WhiteActionButton</item>
-        <item name="android:textColorPrimary">@color/white</item>
```

Comment:
Added this when trying to make actionbar text white. Turns out it makes all text white, like in the addtask activity. White on white isn't great.